src/features/transactions/components/positions-table.tsx

```diff
@@ -1,231 +1,198 @@
 import { useTranslation } from 'react-i18next';
-import { Fragment } from 'react';
-import { ChevronDown, Info } from 'lucide-react';
-import { useQuery } from '@tanstack/react-query';
-import { PositionRow } from './position-row';
-import {
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableHeader,
-  TableRow,
-} from '@/features/shared/components/ui/table';
-import { Skeleton } from '@/features/shared/components/ui/skeleton';
-import { investorsMeTransactionsHistoryListOptions } from '@/client/@tanstack/react-query.gen';
+import { Info } from 'lucide-react';
+import { useState } from 'react';
+import type { ColumnDef, PaginationState } from '@tanstack/react-table';
+import type { HistoryEntry } from '@/client';
 import {
   Tooltip,
   TooltipContent,
   TooltipTrigger,
 } from '@/features/shared/components/ui/tooltip';
-import { Button } from '@/features/shared/components/ui/button';
+import { dateToLocale } from '@/features/shared/utils/date';
+import { Badge } from '@/features/shared/components/ui/badge';
+import { DataTable } from '@/features/shared/components/ui/data-table';
+import { TableCell, TableRow } from '@/features/shared/components/ui/table';
+
+function usePositionsColumns() {
+  const { t, i18n } = useTranslation();
 
-type PositionsTableProps = {
-  type: 'open' | 'closed';
-};
+  const formatCurrency = (value: number): string => {
+    return new Intl.NumberFormat(i18n.language, {
+      style: 'currency',
+      currency: 'USD',
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2,
+    }).format(value);
+  };
```

Comment:
maybe extract this to a separate helper file, i'm positive i've already seen this helper in the positions-cards file? DRY